applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -1040,17 +1036,48 @@ static int cmd_hfclkaudio_drift_comp_disable(const struct shell *shell, size_t a
 	return 0;
 }
 
-SHELL_STATIC_SUBCMD_SET_CREATE(test_cmd,
-			       SHELL_COND_CMD(CONFIG_SHELL, nrf_tone_start, NULL,
-					      "Start local tone from nRF5340.", cmd_i2s_tone_play),
-			       SHELL_COND_CMD(CONFIG_SHELL, nrf_tone_stop, NULL,
-					      "Stop local tone from nRF5340.", cmd_i2s_tone_stop),
-			       SHELL_COND_CMD(CONFIG_SHELL, pll_comp_enable, NULL,
-					      "Enable audio PLL auto drift compensation (default).",
-					      cmd_hfclkaudio_drift_comp_enable),
-			       SHELL_COND_CMD(CONFIG_SHELL, pll_comp_disable, NULL,
-					      "Disable audio PLL auto drift compensation",
-					      cmd_hfclkaudio_drift_comp_disable),
-			       SHELL_SUBCMD_SET_END);
+static int cmd_audio_pres_comp_enable(const struct shell *shell, size_t argc, const char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	ctrl_blk.pres_comp.audio_comp_enabled = true;
+
+	shell_print(shell, "Audio PLL presentation compensation enabled");
+
+	return 0;
+}
+
+static int cmd_audio_pres_comp_disable(const struct shell *shell, size_t argc, const char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	ctrl_blk.pres_comp.audio_comp_enabled = false;
```

Comment:
If the state is e.g. `DRIFT_STATE_LOCKED` and we disable the state machine. Should we by default set the machine to `DRIFT_STATE_INIT` ?

Reply:
Added such that when pres_comp is disabled it sets the state machine to "DRIFT_STATE_INIT"